consumer_base.go

```diff
@@ -129,6 +136,10 @@ func (c *base) startConsume() {
 			close(c.incomingMessageStream)
 			return
 		default:
+			if c.pauseConsuming {
```

Comment:
I checked here Because when a client calls the `Stop` method, I need to listen to `c.quit` and return.